### ISSUE TITLE
Firmware development ongoing?

### ISSUE THREAD

**Issue by SandyMental** [polyend/TrackerBetaTesting#1622]
Or has it been abandoned by Polyend?

Would be kind to get some words from the devs/Polyend on what to expect or not anymore.

**Comment by PRAYTHEPO** [comment]
> Or has it been abandoned by Polyend?
> 
> 
> 
> Would be kind to get some words from the devs/Polyend on what to expect or not anymore.
> 
> 
> 
> 

Last update and contact with Polyend was in July 2022. Answer is obvious

**Comment by SandyMental** [comment]
> > Or has it been abandoned by Polyend?
> > Would be kind to get some words from the devs/Polyend on what to expect or not anymore.
> 
> Last update and contact with Polyend was in July 2022. Answer is obvious

yeah obvious to people lurking around here, but would bet that's not the majority of their  customers ;-)

**Comment by zikajk** [comment]
I had the same question.
I was hoping that the development for Play would also be reflected in Tracker. 
The software seems to be quite similar, so it would be possible to fix the nasty bugs that the Tracker software has. Unfortunately, it seems that the products are not very related and the Tracker will not receive an update for a while :-(. But it is not dead according to Polyend, which is good :-)
Below you can see a reply from Polyend team.


_Thanks for reaching out. I'm sorry to hear you are not satisfied with our product even after the 1.6 firmware update. Tracker development is a bit slower right now since we are concentrating on Play and its growth and as we are a small team we can work on a limited number of projects. There will be a firmware update in the future at some point, but as for now, I can't tell you when exactly. Thank you for understanding._

**Comment by synthexpert** [comment]
> Thanks for reaching out. I'm sorry to hear you are not satisfied with our product even after the 1.6 firmware update. Tracker development is a bit slower right now since we are concentrating on Play and its growth and as we are a small team we can work on a limited number of projects. There will be a firmware update in the future at some point, but as for now, I can't tell you when exactly. Thank you for understanding.

That's reasonable after all. It would just be fine to close all the most annoying bugs...

**Comment by swindus** [comment]
Don't expect anything. I received a similar statement form Polyend regarding development of Medusa firmware and now it's legacy.
